tests/pytest/test.py

```diff
@@ -1414,12 +1414,19 @@ def testRDBUnboundedDepth(env):
     # concat the string_126 at the end of itself
     json_value = nest_object(3, 5, "__deep_leaf", 420)
     r.expect('JSON.SET', 'doc', '$..__leaf', json_value).ok()
-    
+
     # RDB dump and restore the key 'doc' and check that the key is still valid
     dump = env.execute_command('dump', 'doc', **{NEVER_DECODE: []})
     r.expect('RESTORE', 'doc1', 0, dump).ok()
     r.expect('JSON.GET', 'doc1', '$..__leaf..__deep_leaf').equal('[420]')
 
+def testUnicodeCharacters(env):
+    # Test JSONPath filter with unicode characters
+    r = env
+    # Test with some weird unicode characters.
+    r.assertOk(r.execute_command('JSON.SET', 'test', '$', '{"\u00a0": "⍢∪⇰"}'))
+    r.assertOk(r.execute_command('JSON.SET', 'test', '$', '{"\u00a0": "\u00a0\u00a0"}'))
```

Comment:
Can also confirm we can get it back with `JSON.GET`

Reply:
Sure.